app/services/subscriptions/create_service.rb

```diff
@@ -40,10 +40,9 @@ def create(**args)
 
       # NOTE: prepare subscription attributes
       @name = args[:name]&.strip
-      @unique_id = SecureRandom.uuid
+      @external_id = SecureRandom.uuid
```

Comment:
Wondering if this still apply. Should we put the customer external id (Or waiting from the front to provide this value)?

Reply:
Well - Not sure we want the front to manage external ids and not sure we want to store the external customer id.

For me, I think it's good enough to generate this info from the backend, no?